be/src/runtime/mem_tracker.h

```diff
@@ -170,6 +173,12 @@ class MemTracker : public std::enable_shared_from_this<MemTracker> {
             Release(-bytes);
             return true;
         }
+        if (MemInfo::current_mem() + bytes >= MemInfo::mem_limit()) {
+            VLOG_NOTICE << "TryConsume failed, bytes=" << bytes
+                        << " process whole consumption=" << MemInfo::current_mem()
+                        << " mem limit=" << MemInfo::mem_limit();
```

Comment:
We should find a way to make the error more obvious and give a feasible troubleshooting method or handling advice.
Otherwise, this change will simply block the error, but it will not bring more help. (We can't even know if this error has occurred)

Reply:
Can consider printing MemTacker through MemTracker::MemLimitExceeded
P.S. Printing MemTracker::MemLimitExceeded has a certain cost, confirm that it will not be printed frequently.